tracer_scripts/job_start_end.d

```diff
@@ -0,0 +1,44 @@
+#!/usr/sbin/dtrace -s 
```

Comment:
Is this the same as `jobs_start_end.stp`? If yes, please fix the naming and add a short comment.

Reply:
ping

Reply:
Yes, it is the same as `jobs_start_end.stp`. I fixed the naming and added a comment.